src/external/external.js

```diff
@@ -1,12 +1,4 @@
 global.choose = function(className) {
-  const objects = []
-  var classToFind = eval('ObjC.classes.' + className);
-  ObjC.choose(classToFind, {
-    onMatch: function(a) {
-      objects.push(a.toString());
-    },
-    onComplete: function() {
-    }
-  });
-  return objects;;
+  const classToFind = ObjC.classes[className];
+  return ObjC.chooseSync(ObjC.classes[className]).map(instance => instance.toString());
```

Comment:
```suggestion
  return ObjC.chooseSync(classToFind).map(instance => instance.toString());
```

(In hindsight, instead of getting rid of the previous line I figured the variable would help explain the code.)

Reply:
You are right, will commit now.